modules/example-apache-httpd-configuration.adoc

```diff
@@ -7,20 +7,43 @@
 
 The basic identify provider (IDP) configuration in {product-title} 4 requires
 that the IDP server respond with JSON for success and failures. You can use CGI
-scripting in Apache HTTPD to accomplish this. This section provides examples.
+scripting in Apache HTTPD to accomplish this. This section provides examples
+with an httpd server configured as a running pod in cluster.
 
-.Example `/etc/httpd/conf.d/login.conf`
+.Use project "default"
 ----
-<VirtualHost *:443>
+PROJECT=httpd-proj
+APP=httpd
+SVC=basic-auth
+IDP_NAME=basicauth-idp
+oc new-project $PROJECT
+----
+
+.Deploy & Configure the HTTPD Server
+----
+oc get is httpd -n openshift
+
+NAME    IMAGE REPOSITORY                                                   TAGS                         UPDATED
+httpd   image-registry.openshift-image-registry.svc:5000/openshift/httpd   2.4,2.4-el7,2.4-el8,latest   59 minutes ago
+
+oc create deployment $APP --image image-registry.openshift-image-registry.svc:5000/openshift/httpd:2.4 -- bash -c "cp /tmp/httpd/login.conf /etc/httpd/conf.d/ssl.conf; httpd -D FOREGROUND"
+oc expose deployment/$APP --name $SVC --port 8443
+oc annotate service $SVC service.beta.openshift.io/serving-cert-secret-name=httpd-cert
+----
+
+.Use cat > login.conf << EOF
```

Comment:
Now the content of the PR is basically right, but the format is not good. E.g. viewing in https://deploy-preview-43061--osdocs.netlify.app/openshift-enterprise/latest/authentication/identity_providers/configuring-basic-authentication-identity-provider.html#example-apache-httpd-configuration_configuring-basic-authentication-identity-provider , `Use cat > login.conf << EOF` would read strange.
And you'd better place the preparation of the files first, then place other commands.
About formatting review, could you first ask for doc team peer review? After that, then ask me to review. Thanks